console/src/main/webapp/manager/app/app.es6

```diff
@@ -73,18 +73,32 @@ class AppController {
 }
 
 class StandaloneController {
-  static $inject = ['$scope', 'Orgs', 'User']
+  static $inject = ['$injector', '$window', '$http', '$scope', 'Orgs', 'User']
 
-  constructor ($scope, Org) {
-    if (!window.org) {
+  constructor ($injector, $window, $http, $scope, Org) {
+    if (!$window.org) {
       $scope.org = new Org()
       return
     }
 
-    $scope.org = window.org
-    $scope.users = window.org.members
-    $scope.isReferentOrSuperUser = window.isReferentOrSuperUser
-    $scope.gdprAllowAccountDeletion = window.gdprAllowAccountDeletion
+    $scope.org = $window.org
+    $scope.users = $window.org.members
+    $scope.isReferentOrSuperUser = $window.isReferentOrSuperUser
+    $scope.gdprAllowAccountDeletion = $window.gdprAllowAccountDeletion
+    $scope.deleteURI = $injector.get('CONSOLE_BASE_PATH') + 'account/gdpr/delete'
+
+    const i18n = {}
+    $injector.get('translate')('editUserDetailsForm.deleteConfirm', i18n)
+    $injector.get('translate')('editUserDetailsForm.deleteFail', i18n)
```

Comment:
Since you only use $injector for translate, you can inject `translate` instead of `$injector` and use it here.

Reply:
thanks for the info.  Although I'm also using it to get the CONSOLE_BASE_PATH variable. So let's keep it that way ?

Reply:
yep